src/MICore/LaunchOptions.cs

```diff
@@ -513,13 +500,24 @@ public sealed class UnixShellPortLaunchOptions : LaunchOptions
                         startRemoteDebuggerCommand = "lldb-mi --interpreter=mi";
                         break;
                     case MIMode.Clrdbg:
+                        string debuggerHomeDirectory;
+                        if (DebuggerInstallationDirectory.StartsWith("/", StringComparison.OrdinalIgnoreCase))
+                        {
+                            debuggerHomeDirectory = DebuggerInstallationDirectory;
```

Comment:
That is not necessary, because it is gets added later on. Removed the redundant /.